docs/deploying.md

```diff
@@ -150,32 +150,6 @@ OpenStackDataPlaneServices](#create-openstackdataplaneservices) section.
 Common configurations that can be enabled with `ansibleVars` are also
 documented at [Common Configurations](common_configurations.md).
 
-Some of the ansible variables will need to be set based on values from the
-controlplane that is already deployed. This set of ansible variables and the
-`oc` command that can be used to get their values are shown below.
-
-```console
-export EDPM_OVN_METADATA_AGENT_TRANSPORT_URL=$(oc get secret rabbitmq-transport-url-neutron-neutron-transport -o json | jq -r .data.transport_url | base64 -d)
-export EDPM_OVN_METADATA_AGENT_SB_CONNECTION=$(oc get ovndbcluster ovndbcluster-sb -o json | jq -r .status.dbAddress)
-export EDPM_OVN_METADATA_AGENT_NOVA_METADATA_HOST=$(oc get svc nova-metadata-internal -o json |jq -r '.status.loadBalancer.ingress[0].ip')
-export EDPM_OVN_METADATA_AGENT_PROXY_SHARED_SECRET=$(oc get secret osp-secret -o json | jq -r .data.MetadataSecret  | base64 -d)
```

Comment:
this isn't handled in inventory.go though is it? so we'd still need this documented.

Reply:
it made me realize we are getting data from secret and adding to configmap and dataplane CR,
maybe we need to generate a secret instead of a configmap

Reply:
A secret for the inventory? Yes, most likely.